src/components/TranslationBanner.js

```diff
@@ -144,7 +144,7 @@ const TranslationBanner = ({
             </div>
             {!isPageContentEnglish && (
               <div>
-                <SecondaryButtonLink isSecondary to={`${originalPagePath}`}>
+                <SecondaryButtonLink isSecondary to={`${"/en" + originalPagePath}`}>
```

Comment:
```suggestion
                <SecondaryButtonLink isSecondary to={`/en${ originalPagePath}`}>
```
Just small suggestion to clean up code